homepage/models.py

```diff
@@ -40,25 +44,35 @@ class Req_Code(models.IntegerChoices):
     req_code = models.SmallIntegerField(choices=Req_Code.choices, default=Req_Code.NONE)
 
 
-class User(models.Model):
-    name = models.CharField(max_length=100)
-    email = models.CharField(max_length=50)
-    password = models.CharField(max_length=30)
+class AppUser(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+
+    def __str__(self):
+        return self.user.username
 
 
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(AppUser, on_delete=models.CASCADE)
     rate = models.SmallIntegerField()
     date = models.DateTimeField(default=timezone.now)
     content = models.TextField()
     course_load = models.SmallIntegerField()
 
+    def __str__(self):
+        return 'Preview: {}'.format(self.content[:20])

```

Comment:
Is it not critical if it cuts off a word in the middle?
If it is, can [this](https://stackoverflow.com/questions/31978086/set-a-limit-to-the-number-of-words-or-characters-in-a-string) be implemented?

Reply:
From what i understand - you mean cutting the string into segments of words so the word will not be cut in the middle?

Sounds like a good idea, i changed the method and squashed the previous commit.

Reply:
Exactly :)